test/run.py

```diff
@@ -28,5 +28,11 @@
 if not sys.argv[1].startswith('plugin://'):
     sys.argv[1] = 'plugin://plugin.video.vrt.nu' + sys.argv[1]
 
-print('** Running URI: %s' % sys.argv[1])
-addon.run([sys.argv[1], 0, ''])
+# Split path and args
+try:
+    path, args = sys.argv[1].split('?', 1)
```

Comment:
So I really like exception handling in general, but try to avoid it in those cases where it is easy to avoid.
And the main reason is that exception handling has a performance impact (not that it would matter here)

This would do:
```python
path, _, args = sys.argv[1].partition('?')
```

Reply:
I'll modify and merge.